recipe/set_bld_opts.bat

```diff
@@ -62,7 +62,7 @@ set BLD_OPTS=%WIN64% ^
     NETCDF_LIB=%LIBRARY_LIB%\netcdf.lib ^
     NETCDF_SETTING=yes ^
     NETCDF_HAS_NC4=yes ^
-    OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-%openjpeg%" ^
+    OPENJPEG_CFLAGS="-I%LIBRARY_INC%\openjpeg-2.5" ^
```

Comment:
Happy to go back to something more general if there's a way.

Reply:
We loosened the pin on openjpeg, so the old trick doesn't work.one would have to search for the installed version instead. Not sure if pkgconfig can be used instead.

Reply:
Or if it is cake based, a findpkg would be good to add upstream

Reply:
No, not yet cmake based.  That didn't go well, see #651

Reply:
I personally don't think the pkgconfig approach is worth the trouble and we should hard-code this until CMake can help later on.